web-ui/yarn.lock

```diff
@@ -1363,7 +1363,7 @@
   dependencies:
     eslint-visitor-keys "^3.3.0"
 
-"@eslint-community/regexpp@^4.6.1":
+"@eslint-community/regexpp@^4.10.0", "@eslint-community/regexpp@^4.6.1":
```

Comment:
Are we using two versions of this same package?

Reply:
We're not, but one of our transient dependencies may be.